targets/targets.json

```diff
@@ -1111,9 +1111,8 @@
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM"],
         "program_cycle_s": 2,
         "extra_labels": ["STM", "STM32F4", "STM32F407", "STM32F407xG", "STM32F407VG"],
-        "macros": ["LSI_VALUE=32000"],
+        "macros": ["LSI_VALUE=32000", "TRANSACTION_QUEUE_SIZE_SPI=2"],
```

Comment:
This isn't just a reformatting change. Did you intend for this to go in as well ?